code/modules/client/preferences.dm

```diff
@@ -1065,7 +1065,7 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 						age = max(min( round(text2num(new_age)), AGE_MAX),AGE_MIN)
 				if("species")
 
-					var/list/new_species = list("Human", "Tajaran", "Skrell", "Unathi", "Diona", "Vulpkanin")
+					var/list/new_species = list("Human", "Tajaran", "Skrell", "Unathi", "Diona", "Vulpkanin", "Drask")
 					var/prev_species = species
 //						var/whitelisted = 0
 

```

Comment:
This file has a lot of species checks in the `ShowChoices` proc, you may want to make sure, in particular, that Drask players are able to change body color, if you want that something to be changeable.